Neos.Neos/Classes/AssetUsage/Service/AssetUsageSyncService.php

```diff
@@ -56,6 +56,7 @@ public function isAssetUsageStillValid(AssetUsage $usage): bool
         $dimensionSpacePoint = $usage->originDimensionSpacePoint->toDimensionSpacePoint();
 
         $subGraph = $this->contentGraph->getSubgraph(
+            // todo use workspace name instead!!!
             $usage->contentStreamId,
```

Comment:
i guess adding the `workspace` to the `AssetUsage` will not be trivial, so we should just use the workspace finder beforehand and get a workspace that will be internally resolved again to a content stream?